ssh_ldap_pubkey/__init__.py

```diff
@@ -84,6 +84,9 @@ def connect(self):
             conn.network_timeout = conf.bind_timeout
             conn.timeout = conf.search_timeout
 
+            if conf.ssl == 'start_tls' and conf.uri.startswith('ldap://'):
```

Comment:
Why do you explicitly check if the URI starts with `ldap://`?

According to [python-ldap documentation](https://www.python-ldap.org/doc/html/ldap.html#ldap.LDAPObject.start_tls_s), StartTLS requires protocol version 3, so we should check if `conf.ldap_version` is ≥ 3.